tests/api/api.spec.ts

```diff
@@ -1366,6 +1367,28 @@ postOps`
       expect(result.actions[0].tasks[0].errorMessage).to.match(/cancelled/);
     });
   });
+
+  suite("error_parsing", () => {
```

Comment:
why not move this case to each of the integration tests, then we'll be testing against the actual data that snowflake/BQ/whatever returns?